opm/simulators/flow/BlackoilModelEbos.hpp

```diff
@@ -27,6 +27,24 @@
 #include <ebos/eclproblem.hh>
 #include <opm/models/utils/start.hh>
 
+#include <opm/simulators/timestepping/AdaptiveTimeSteppingEbos.hpp>
```

Comment:
Tiny nit: We also include this header [a few lines down](https://github.com/OPM/opm-simulators/blob/424ee2174d1f69ddc082f66881d9b73f739a2782/opm/simulators/flow/BlackoilModelEbos.hpp#L67).  It doesn't really matter since we have [include guards](https://github.com/OPM/opm-simulators/blob/a31f1cefd527c20ff6b35fdc5aa8622a07488725/opm/simulators/timestepping/AdaptiveTimeSteppingEbos.hpp#L3-L4), but maybe we could remove one of the include statements?